src/log/log_test.go

```diff
@@ -0,0 +1,38 @@
+//go:build unit
+// +build unit
+
+package log_test
+
+import (
+	"testing"
+
+	"github.com/NeowayLabs/semantic-release/src/log"
+	"github.com/NeowayLabs/semantic-release/src/tests"
+)
+
+func TestNewErrorLevel(t *testing.T) {
+	_, err := log.New("test", "0.1", "not-found")
+	tests.AssertError(t, err)
+}
+
+func TestNewNoError(t *testing.T) {
```

Comment:
why didn't you call any log print function? @esequielvirtuoso

Reply:
Done.

Reply:
@LeoCBS  Can you check that once again?